smallapp/unbound-anchor.c

```diff
@@ -187,7 +187,7 @@ usage(void)
 	printf("-c file		cert file, default %s\n", ROOT_CERT_FILE);
 	printf("-l		list builtin key and cert on stdout\n");
 	printf("-u name		server in https url, default %s\n", URLNAME);
-	printf("-S		use SNI for the https connection\n");
+	printf("-S		do not use SNI for the https connection\n");
```

Comment:
Still have to document -S in doc/unbound-anchor.8.in.  In the man page for the commandline options for unbound-anchor.